<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Система управления транспортом</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_title">
      <property name="text">
       <string>Система управления транспортом</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_transport_type">
        <property name="text">
         <string>Тип транспорта:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_transport_type">
        <item>
         <property name="text">
          <string>Грузовик</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Корабль</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Самолет</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Дрон</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Поезд</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_distance">
        <property name="text">
         <string>Расстояние (км):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_distance">
        <property name="placeholderText">
         <string>Введите расстояние</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_cargo">
        <property name="text">
         <string>Вес груза (тонн):</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_cargo">
        <property name="placeholderText">
         <string>Введите вес груза</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
      <item>
       <widget class="QPushButton" name="button_calculate">
        <property name="text">
         <string>Рассчитать доставку</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_generate_report">
        <property name="text">
         <string>Создать отчет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
       <widget class="QTextEdit" name="textEdit_output">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
	  </item>
     <item>
       <widget class="QTextEdit" name="textEdit_output_report">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
	  </item>
     </layout>
    </item>
      <item>
       <widget class="QPushButton" name="button_clear">
        <property name="text">
         <string>Очистить расчеты</string>
        </property>
       </widget>
      </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
